StringArt/StringArt.js

```diff
@@ -0,0 +1,128 @@
+import { Polar } from "../sketchlib/Polar.js";
+
+const MODULUS = 60;
+
+/**
+ * Polynomial ax^3 + bx^2 + cx + d where the coefficients
+ * and values are evaluated mod n
+ */
+class Polynomial {
+  constructor(a, b, c, d) {
+    this.coefficients = [a, b, c, d];
+  }
+
+  compute(x) {
+    const x2 = (x * x) % MODULUS;
```

Comment:
The modulus should be a member variable to make this class more reusable. That said, this sketch should still pick a constant one.